pkg/volume/flexvolume/plugin.go

```diff
@@ -62,7 +62,7 @@ func (plugin *flexVolumePlugin) getExecutable() string {
 
 // Name is part of the volume.VolumePlugin interface.
 func (plugin *flexVolumePlugin) GetPluginName() string {
-	return plugin.driverName
+	return "flexvolume-" + plugin.driverName
```

Comment:
Not sure if there'd be an issue with two '/' s, since vendor and driver are separated by a slash in driverName as well. /cc @chakri-nelluri